tests/getOrInitializeBotData.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * tests/getOrInitializeBotData
+ */
+
+const expect = require('chai').expect;
+const rewire = require('rewire');
+const serialize = require('serialize-javascript');
+const config = { refocusUrl: 'zzz', token: 'dummy' };
+const bdk = rewire('../refocus-bdk-client.js');
+const roomId = 10;
+const botName = 'test';
+
+global.user = '{&quot;email&quot;:&quot;test@test.com&quot;}';
+global.window = { document: { }, location: { href: '' } };
+
+describe('getOrInitializeBotData function >', () => {
+  it('Ok, BotData exists, just retrieve it', (done) => {
+    const testStr = 'testing';
```

Comment:
As a good practice, we can use expectedResponse or expected{...Something descriptive}